config/sync/config_ignore.settings.yml

```diff
@@ -1,4 +1,5 @@
 ignored_config_entities:
   - 'system.site:page.front'
+  - '*block.block.footerplayland.yml'
```

Comment:
You don't need to add the `.yml` ending to the config, and the asterisk is not necessary either because you specified the full name of the config.